spec/features/conference_registration_spec.rb

```diff
@@ -49,5 +65,16 @@
         expect(conference.user_registered?(participant)).to be(true)
       end
     end
+
+    context 'registration is closed' do
+      let(:conference_with_closed_registration) { create(:conference, registration_period: create(:registration_period)) }
+
+      scenario 'registers for a conference', feature: true do
+        participant.is_admin = false
```

Comment:
Chekout #799 I think it is know issue that first user is admin in text env too. You guys have normally created a `first_user` in other specs to get around this.

I have to use it cause, even `create(:user, is_admin: false)` make no difference :sweat: